### PR TITLE
Compile Django message files and don't add gettext to slug

### PR DESCRIPTION
**Pull request by codingjoe**

### CODE REVIEW

File: bin/compile [hunk 1]
```diff
@@ -48,20 +42,12 @@ if [ ! `which msgfmt` ]; then
...
     exit 1
 fi
 
-mkdir -p $BUILD_DIR/.profile.d
-cat <<EOF >$BUILD_DIR/.profile.d/gettext.sh
-export PATH=$HOME/.gettext/bin:$PATH
-export LD_LIBRARY_PATH=$HOME/.gettext/lib:$LD_LIBRARY_PATH
-EOF
+build-step "run msgfmt"
+find $BUILD_DIR -name \*.po -execdir msgfmt django.po -o django.mo \;
```
Comment (syphar) [inline bin/compile:53]: @codingjoe IMHO this buildpack should only take care of getting gettext into the slug/cache, nothing else.

File: bin/compile [hunk 2]
```diff
@@ -7,24 +7,18 @@ BUILD_DIR=$1
...
 BIN_DIR=$(cd $(dirname $0); pwd) # absolute path
 
-GETTEXT_TAR="http://ftp.gnu.org/pub/gnu/gettext/gettext-0.18.3.1.tar.gz"
-GETTEXT_HOME=$BUILD_DIR/.gettext
+GETTEXT_TAR="http://ftp.gnu.org/pub/gnu/gettext/gettext-0.19.8.1.tar.gz"
 GETTEXT_CACHE=$CACHE_DIR/.gettext
 
-PATH=$GETTEXT_HOME/bin:$PATH
-LD_LIBRARY_PATH=$GETTEXT_HOME/lib:$LD_LIBRARY_PATH
+PATH=$GETTEXT_CACHE/bin:$PATH
```
Comment (syphar) [inline bin/compile:13]: perhaps this should be a env/setting somewhere? 

I can imagine people using this buildpack and wanting to have the binaries

Comment (syphar) [reply bin/compile:13]: apart from that, it makes it backwards incompatible

**Review by syphar — CHANGES_REQUESTED** [review]

**Comment by codingjoe** [comment]
@syphar it's definitely backwards incompatible, in facts it's a very different idea. it's only for Django and it's not a gettext build pack, but a gettext compiler. I guess it makes more sense to create an entirely new build pack.

**Comment by syphar** [comment]
@samifahed I do not believe this will give us enough savings

**Pull request closed without merging**